.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+Please include a summary of the changes and the related issue. List any dependencies that are required for this change.
+
+Fixes #(number of issue in github)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
```

Comment:
I would remove this section. Tests should be added for all functionality anyways, and we don't have the need for manual testing (unlike for example, in the Aspect Model Editor). Also we don't care for firmware or hardware.